framework/_mixins.scss

```diff
@@ -255,43 +255,6 @@ $modifiers: (
   }
 }
 
-// Should be used only with @extend.
-.icon--after {
-  display: inline-block;
-  margin-left: 4px;
-  line-height: 1;
-  font-family: 'EuropaIcons';
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  // Better Font Rendering
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-transform: none;
-  text-decoration: underline;
-  speak: none;
-}
-
-// Remove the underline from a::after
-// There is a bug in IE8-11, so using display:inline-block; alone won't
-// work. The solution is by explicitly setting text-decoration:underline;
-// for the ::after-content and then overwrite this rule again with
-// text-decoration:none;
-.icon--after {
-  text-decoration: none;
-}
-
-// [converter] $parent hack.
-@mixin hidden-print($selector) {
-  #{$selector} {
-    display: none !important;
-  }
-}
-
-@media print {
```

Comment:
Sure! First, the `.hidden-print` class already exists in Bootstrap, there's no need to re-implement it. Then, since it's recommended to only extend placeholders and not classes, I added the %hidden-print placeholder here: https://github.com/ec-europa/europa-component-library/blob/1749ac739c0a8a6d89a9ab5394d2e54ba49854a5/framework/_global.scss#L19 (I'm not sure where it fits the best but _global.scss sounds okay)